downstream/modules/automation-hub/proc-run-setup-script-hub.adoc

```diff
@@ -0,0 +1,14 @@
+[id="run-setup-script-hub"]
+
+= Running the setup script
+
+You can run the setup script once you finish updated the `inventory` file with required parameters for installing your Private Automation Hub.
```

Comment:
when you finish updating the `inventory file` with the required parameters for installing your {PrivateHubName}.